packages/select/src/option-group.vue

```diff
@@ -37,8 +38,32 @@ export default defineComponent({
 
     const select = inject(selectKey)
 
+    onMounted(() => {
+      children.value = flattedChildren(instance.subTree)
+    })
```

Comment:
I have been looking for a long time, but I have no idea how to get the options under the current group, I learned it

Reply:
Although I think this is not the best way or the best approach to solve this issue you can see we took different approach in `ElSelectV2` which used data with explicit structure that hints us to render group or item. 

AntDesign took both ways for the users which I think the structured data would be more elegant.